frontend/.env.production

```diff
@@ -1,3 +0,0 @@
-VUE_APP_FRONTEND=https://www.theurlist.com

```

Comment:
Should this file be deleted?

Reply:
Yeah - we shouldn't have any environment keys in the source.